pkg/controller/migcluster/validation.go

```diff
@@ -159,7 +148,7 @@ func (r ReconcileMigCluster) validateSaSecret(cluster *migapi.MigCluster) error
 			Status:   True,
 			Reason:   NotFound,
 			Category: Critical,
-			Message:  InvalidSaSecretRefMessage,
+			Message:  fmt.Sprintf("The `serviceAccountSecretRef` must reference a valid `secret`, subject %s", path.Join(cluster.Spec.ServiceAccountSecretRef.Namespace, cluster.Spec.ServiceAccountSecretRef.Name)),
```

Comment:
We try to keep line length <= 100.  Mainly to keep the code easy to view in github.

Reply:
Also, don't remember the convention you followed in the other PR but thinking that `subject:` may be clearer.  (with colon).

Reply:
Agreed, updated the PR.